src/main/resources/entity-types/pol/drv_purchase_order_line_details.json5

```
@@ -3,6 +3,42 @@
   name: 'drv_purchase_order_line_details',
   private: false,
   fromClause: "src_purchase_order_line JOIN src_purchase_order purchase_order ON purchase_order.id = ((src_purchase_order_line.jsonb ->> 'purchaseOrderId'::text)::uuid) LEFT JOIN src_users_users user_details ON user_details.id = (((purchase_order.jsonb -> 'metadata'::text) ->> 'createdByUserId'::text)::uuid) LEFT JOIN src_users_users user_details_for_order_updated_by ON user_details_for_order_updated_by.id = (((purchase_order.jsonb -> 'metadata'::text) ->> 'updatedByUserId'::text)::uuid) LEFT JOIN src_users_users user_details_for_pol_created_by ON user_details_for_pol_created_by.id = (((src_purchase_order_line.jsonb -> 'metadata'::text) ->> 'createdByUserId'::text)::uuid) LEFT JOIN src_users_users user_details_for_pol_updated_by ON user_details_for_pol_updated_by.id = (((src_purchase_order_line.jsonb -> 'metadata'::text) ->> 'updatedByUserId'::text)::uuid) LEFT JOIN src_organizations organization_details ON organization_details.id = ((purchase_order.jsonb ->> 'vendor'::text)::uuid) LEFT JOIN src_users_users user_details_of_assignee ON user_details_of_assignee.id = ((purchase_order.jsonb ->> 'assignedTo'::text)::uuid) LEFT JOIN currency_exchange_rates rates ON rates.currency = src_purchase_order_line.jsonb -> 'cost' ->> 'currency'",
+  requiredPermissions: [ // These are the sub-permissions of ui-orders.orders.view
+    "module.orders.enabled",
+    "addresstypes.collection.get",
+    "acquisition.order.events.get",
+    "acquisition.order-line.events.get",
+    "finance.calculate-exchange.item.get",
+    "finance.budgets.collection.get",
+    "finance.exchange-rate.item.get",
+    "finance.expense-classes.collection.get",
+    "finance.fiscal-years.collection.get",
+    "finance.funds.expense-classes.collection.get",
+    "inventory.instances.collection.get",
+    "orders.export-history.collection.get",
+    "orders-storage.order-invoice-relationships.collection.get",
+    "orders.acquisition-method.item.get",
+    "orders.acquisition-methods.collection.get",
+    "orders.collection.get",
+    "orders.configuration.prefixes.collection.get",
+    "orders.configuration.reasons-for-closure.collection.get",
+    "orders.configuration.suffixes.collection.get",
+    "orders.item.get",
+    "orders.order-templates.item.get",
+    "orders.pieces.collection.get",
+    "orders.po-lines.collection.get",
+    "orders.po-lines.item.get",
+    "orders.routing-lists.collection.get",
+    "orders.routing-lists.item.get",
+    "orders.titles.collection.get",
+    "organizations.organizations.collection.get",
+    "organizations-storage.organization-types.collection.get",
+    "ui-orders.third-party-services",
+    "ui-orders.custom-fields.view",
+    "consortia.publications.item.get",
+    "consortia.publications.item.post",
+    "consortia.publications-results.item.get"
```

```suggestion
    "orders.po-lines.item.get",
    "orders.item.get",
    "acquisitions-units.units.collection.get",
    "organizations.organizations.item.get",
    "users.collection.get",
    "finance.exchange-rate.item.get",
    "configuration.entries.collection.get"
```

Here's the minimum set of permissions required to access the data used by the POL entity type